cloud-consumption-api/src/main/java/com/sequenceiq/consumption/api/v1/consumption/model/request/StorageConsumptionRequest.java

```diff
@@ -0,0 +1,49 @@
+package com.sequenceiq.consumption.api.v1.consumption.model.request;
+
+import javax.validation.constraints.NotNull;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.sequenceiq.cloudbreak.auth.crn.CrnResourceDescriptor;
+import com.sequenceiq.cloudbreak.validation.ValidCrn;
+import com.sequenceiq.consumption.api.doc.ConsumptionModelDescription;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+@ApiModel(value = "StorageConsumptionRequest")
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class StorageConsumptionRequest extends ConsumptionBaseRequest {
```

Comment:
Just a discussion-opening question: is it the same request class for both schedule and unschedule.
Also, the naming is not consistent with the endpoint method names where this class is used as parameter type:  StorageUsage vs StorageConsumption

Reply:
Good point, created different requests for the two actions and made the method/class naming consistent